code/modules/mining/lavaland/necropolis_chests.dm

```diff
@@ -762,8 +762,8 @@
 /obj/item/melee/transforming/cleaving_saw
 	name = "cleaving saw"
 	desc = "This saw is the tool of choice for the Head Hunter. Capable of switching its reach and attack speed on the fly, it's an incredibly useful weapon for slaying the denizens of the wastes. Animal or human, the saw doesn't judge."
-	force = 30
-	force_on = 60 //force when active
+	force = 25
```

Comment:
I'd recommend swapping these numbers around, so that it does more damage while folded up, and less damage while extended, if I'm reading this correctly. Extra reach is *powerful* after all~

Reply:
fixed